browser/css/calculator.css

```diff
@@ -0,0 +1,97 @@
+@font-face {
+  font-family: "Roboto", sans-serif;
+}
+
+* {
+  font-size: 100%;
```

Comment:
I've never needed to assign `font-size: 100%;` to all elements. What prompted this style?